pkgs/by-name/gy/gysmo/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, go }:
+
+stdenv.mkDerivation rec {
+  pname = "gysmo";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "grosheth";
+    repo = "gysmo";
+    rev = "v${version}";
```

Comment:
`rev` -> `tag`, `sha256` -> `hash`. BTW, you can set `hash = "";` and run `nix build .#gysmo` to get a hash string.